src/vs/platform/backup/electron-main/backupMainService.ts

```diff
@@ -51,11 +51,6 @@ export class BackupMainService implements IBackupMainService {
 	}
 
 	public registerWindowForBackups(windowId: number, isEmptyWorkspace: boolean, backupFolder?: string, workspacePath?: string): void {
```

Comment:
@Tyriar I didnt know you had this check in here, I am also fine leaving it there to keep this knowledge inside the service!